pymc/distributions/continuous.py

```diff
@@ -1642,6 +1642,15 @@ def logp(value, b, kappa, mu):
             msg="b > 0, kappa > 0",
         )
 
+    def icdf(p, mu, b, kappa):
+        if b <= 0:
+            raise ValueError("Scale Parameter sigma must be positive")
```

Comment:
What's `sigma`?

Reply:
Oh, sorry. Actually the resource from where I was referring the icdf equation, there they used sigma instead of b. I wrote it out of mistake. I will correct that one and change it to b